code/modules/projectiles/guns/projectile/automatic/lmg.dm

```diff
@@ -135,4 +137,4 @@
 	result = /obj/item/gun/projectile/automatic/lmg/tk
 	grip = /obj/item/part/gun/grip/rubber
 	mechanism = /obj/item/part/gun/mechanism/machinegun
```

Comment:
```suggestion
	gripvars = /obj/item/part/gun/grip/rubber
	mechanismvar = /obj/item/part/gun/mechanism/machinegun
```

Reply:
Not sure why it wouldn't let me suggest for all three lines, but here's a solution for the conflict

Reply:
Make them into lists